xdem/dem.py

```diff
@@ -105,6 +107,20 @@ def set_vref(self,vref_name=None,vref_grid=None,compute_ccrs=False):
         :return:
         """
 
+        # temporary fix for some CRS with proj < 7.2
```

Comment:
You can check the proj version using:
```python
if pyproj.proj_version_str >= "7.2.0":
    # do stuff
```

Reply:
just pushed the change!

Reply:
I didn't know it was possibly to just compare the strings! I thought the comparison would be based on the number of characters.